### ISSUE TITLE
License

### ISSUE THREAD

**Issue by BMillsVT** [timjroberts/cucumber-js-tsflow#32]
What license applies to this library? Please add the license text to the repo so that I can use it at work.

**Comment by mikehaas763** [comment]
@timjroberts this issue was spurred from me putting in a request to my company's open source office requesting permission to contribute to this project. Would you be ok with it if I added a license? Any particular license you have in mind? MIT?

**Comment by timjroberts** [comment]
@mikehaas763 Yeah, MIT would be good and it's quite commonly supported across corporate commissions with open source.

That work for you @BMillsVT?

**Comment by BMillsVT** [comment]
That would be great from my point of view. Thanks guys.
